src/Entity/Tags.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use ApiPlatform\Metadata\ApiResource;
+
+#[ORM\Entity]
+#[ApiResource]
+class Tags
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
```

Comment:
Could you use Uuid instead as done in all entities?

Reply:
Yes i can but i don't like use type string for a ID

Reply:
> Yes i can but i don't like use type string for a ID

Ok, why? :)

Reply:
Because I learn and I read this.  It's more optimized for "join" and needs less space. UUID.It's good for security for user's interface. You can use both .

Reply:
> optimized for "join"

We don't have a lot of data here so it's not a problem and we don't have a lot of relations between objects

> needs less space

Even is UUID is stored as binary?

> user's interface

UUID for user interface?

Reply:
> We don't have a lot of data here so it's not a problem and we don't have a lot of relations between objects
 Well so do nothing. After executing the command, I have ~2000 tuples, I agree, it's not a lot. 
> Even if UUID is stored as binary?
It seems to me that it's not the most important for me.
> UUID for user interface?
UUID use in href better than an ID for example